benefits/settings.py

```diff
@@ -84,7 +84,7 @@ def _filter_empty(ls):
 # only used if enabled above
 OPENCENSUS = {
     "TRACE": {
-        "SAMPLER": "opencensus.trace.samplers.ProbabilitySampler(rate=1)",
+        "SAMPLER": "opencensus.trace.samplers.AlwaysOnSampler()",
```

Comment:
Can you explain the reason for this change? I'm having trouble finding documentation on the differences between these samplers / what they are meant for

Reply:
I [stumbled upon it in the docs](https://github.com/census-instrumentation/opencensus-python/blob/master/docs/trace/usage.rst#samplers). Basically, if you have a lot of traffic, you don't want to be sending data to Azure every time. The `ProbabilitySampler` allows you to specify a ratio. Since we had `rate=1`, it's the same as `AlwaysOn`.